app/models/alaveteli_pro/draft_info_request_batch.rb

```diff
@@ -43,7 +47,9 @@ def request_summary_body
 
   # @see RequestSummaries#request_summary_public_body_names
   def request_summary_public_body_names
-    self.public_bodies.pluck(:name).join(" ")
+    self.public_bodies.
+      joins(:translations).
```

Comment:
Shouldn't globalize make this join? If not then possibly want a spec & definitely an explanation of why we need this and what's breaking in the commit message.